UIPS/UIP-0122.md

```diff
@@ -0,0 +1,93 @@
+---
+uip: "0122"
+title: "%wild: Stateless jet registration"
+description: Propose a new system for deriving jet registrations and hinting which requires no stored state in the interpreter.
+author: ~ritpub-sipsyl (eamsden) 
+status: Draft
+type: Standards Track
+category: Hoon, Nock # Only required for Standards Track. Otherwise, remove this field.
+created: 2024-03-25
+---
+
+## Abstract
+
+This proposal introduces a stateless approach to jet registration in Nock interpreters, utilizing a dynamic hint labeled `%wild`. The `%wild` hint contains the necessary slice of "cold state" for the hinted Nock, thereby avoiding issues with state maintenance and preservation. This approach offers a simple solution to the problems of state in jet registration.
+
+The `%wild` hint can be composed by Hoon and can be implemented using existing information in the subject type, extended only by labels from existing `%fast`-hint runes. It also provides a scoping mechanism which prevents user code from altering kernel code registrations.
+
+An additional advantage of this  approach is its potential to fix the jet mismatch of `+mink`, without requiring any injection of state.
```

Comment:
what jet mismatch in `+mink`? stack trace determinism, or something else?

Reply:
yes stack trace determinism. Will clarify.